.gitignore

```diff
@@ -106,6 +105,7 @@ uspace/app/fdisk/fdisk
 uspace/app/fontviewer/fontviewer
 uspace/app/getterm/getterm
 uspace/app/gunzip/gunzip
+uspace/app/hbench/hbench
```

Comment:
Does 'h' stand for HelenOS? Please do NOT encode name of the project into names of any binaries, tools, etc. What if the name ever needs to change or the tool is ported to another OS? Can we have e.g. just bench?